fuzz/src/chanmon_consistency.rs

```diff
@@ -734,6 +734,12 @@ pub fn do_test(data: &[u8]) {
 				nodes[2] = node_c.clone();
 				monitor_c = new_monitor_c;
 			},
+			0x22 => send_payment!(nodes[0], (&nodes[1], chan_a), 20),
+			0x23 => send_payment!(nodes[1], (&nodes[0], chan_a), 20),
+			0x25 => send_payment!(nodes[1], (&nodes[2], chan_b), 20),
+			0x26 => send_payment!(nodes[2], (&nodes[1], chan_b), 20),
+			0x27 => send_payment!(nodes[0], (&nodes[1], chan_a), (&nodes[2], chan_b), 20),
+			0x28 => send_payment!(nodes[2], (&nodes[1], chan_b), (&nodes[0], chan_a), 20),
```

Comment:
We're not (yet) short on action byte values, to let the fuzzer search a bit better, it may make sense to define a few buckets, and eg have a 5_000_000, a 100_000, a 1_000, and a 10 or something?